src/app/shared/components/accordion/accordion.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="ecl-accordion" data-ecl-auto-init="Accordion" data-ecl-accordion>
```

Comment:
We have the eUI angular components, why are we hardcoding the ECL markup in our components?
Accordion is <ecl-accordion>, the items are <ecl-accordion-items>, we should use this markup to benefit from the existing functionalities associated to the component and also because otherwise we would lose the advantage of using a libray that has an implementation of the ECL vanilla components.

Reply:
I am using now the eui ecl components. thanks.